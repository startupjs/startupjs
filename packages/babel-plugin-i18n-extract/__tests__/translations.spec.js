const fs = require('fs')
const path = require('path')

it('Check if the translations.json is correct', async () => {
  await new Promise((resolve, reject) => {
    setTimeout(() => {
      resolve()
    }, 3000)
  })

  const filePath = path.join(__dirname, '../translations.json')
  const content = fs.readFileSync(filePath, { encoding: 'utf8' })
  expect(content).toMatchSnapshot()
})
