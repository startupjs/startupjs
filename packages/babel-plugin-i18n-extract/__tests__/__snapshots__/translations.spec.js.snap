// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Check if the translations.json is correct 1`] = `"{\\"correct.js\\":{\\"key\\":\\"defaultValue\\"}}"`;
